src/vs/workbench/browser/workbench.contribution.ts

```diff
@@ -306,7 +306,19 @@ import { isMacintosh, isWindows, isLinux, isWeb } from 'vs/base/common/platform'
 				'scope': ConfigurationScope.APPLICATION,
 				'markdownDescription': nls.localize('disableCustomMenuBarAltFocus', "If enabled, disables the ability to focus the menu bar with the Alt-key when not set to toggle."),
 				'included': isWindows || isLinux || isWeb
-			}
+			},
+			'window.openFoldersInNewWindow': {
+				'type': 'string',
+				'enum': ['on', 'off', 'default'],
+				'enumDescriptions': [
+					nls.localize('window.openFoldersInNewWindow.on', "Folders will open in a new window."),
+					nls.localize('window.openFoldersInNewWindow.off', "Folders will replace the last active window."),
+					nls.localize('window.openFoldersInNewWindow.default', "Folders will open in a new window unless a folder is picked from within the application (e.g. via the File menu).")
+				],
+				'default': 'default',
```

Comment:
do we want to make the default to always open in new window on web?

Reply:
Let's handle this in a follow up with @bpasero's review.

Reply:
https://github.com/microsoft/vscode/pull/75868